orion-viewer/src/main/java/universe/constellation/orion/viewer/FileChooser.java

```diff
@@ -46,18 +46,35 @@
 
     private FilenameFilter filter;
 
+    private static Set<String> extensions = new HashSet<>(Arrays.asList("cbz","djvu","djv","pdf","oxps","tif","tiff","xps"));
+
+
     public static FilenameFilter DEFAULT_FILTER = new FilenameFilter() {
-        public boolean accept(File dir, String filename) {
-        if (new File(dir, filename).isDirectory()) {
-            return true;
+        private String getExtension(String filename) {
+            for (int i = filename.length() - 1; i >= 0; i--) {
+                char x = filename.charAt(i);
+                if (x == '.') {
+                    return filename.substring(i + 1);
+                }
+            }
+            return filename;
         }
-        String name = filename.toLowerCase();
-        return name.endsWith(".pdf") || name.endsWith(".djvu") || name.endsWith(".djv") || name.endsWith(".xps") || name.endsWith(".oxps") || name.endsWith(".cbz")
-                || name.endsWith(".tiff") || name.endsWith(".tif") /*|| name.endsWith(".png") || name.endsWith(".jpeg") || name.endsWith(".jpg")*/;
+
+        public boolean accept(File dir, String filename) {
+            if (new File(dir, filename).isDirectory()) {
+                return true;
+            }
+            String name = filename.toLowerCase();
```

Comment:
It's non necessary to call 'toLowerCase' on filename, call it on extension would be enough